frontend/src/pages/auth/signup/SignUp.tsx

```diff
@@ -82,62 +83,64 @@ const SignUp: React.FC = () => {
-                    defaultEmail={userCredentials.email}
-                    updateEmailFields={updateEmailChange}
-                />
-                <NameField
-                    defaultName={userCredentials.name}
-                    updateNameFields={updateNameChange}
-                />
-                <PasswordField
-                    updatePasswordFields={updatePasswordChange}
-                />
-                <Typography variant="h5" gutterBottom>
-                    {Titles.SELECT_ROLES}
-                </Typography>
-                <RadioGroup
-                    row
-                    className="role-selector"
-                    value={selectedRole}
-                    onChange={handleRoleChange}
-                >
-                    <Box className="role-option">
-                        <FormControlLabel value={UserType.Customer} control={<Radio/>} label={Roles.CUSTOMER}/>
-                    </Box>
-                    <Box className="role-option">
-                        <FormControlLabel value={UserType.Merchant} control={<Radio/>} label={Roles.MERCHANT}/>
+        <Box className="main-container">
+            <Box className="content-container">
+                <Card className="card-background card-payment card-auth">
+                    <AuthLogo/>
+                    <EmailField
+                        defaultEmail={userCredentials.email}
+                        updateEmailFields={updateEmailChange}
+                    />
+                    <NameField
+                        defaultName={userCredentials.name}
+                        updateNameFields={updateNameChange}
+                    />
+                    <PasswordField
+                        updatePasswordFields={updatePasswordChange}
+                    />
+                    <Typography variant="h5" gutterBottom>
+                        {Content.SELECT_ROLES}
+                    </Typography>
+                    <RadioGroup
+                        row
+                        className="role-selector"
+                        value={selectedRole}
+                        onChange={handleRoleChange}
+                    >
+                        <Box className="role-option">
+                            <FormControlLabel value={UserType.Customer} control={<Radio/>} label={Roles.CUSTOMER}/>
+                        </Box>
+                        <Box className="role-option">
+                            <FormControlLabel value={UserType.Merchant} control={<Radio/>} label={Roles.MERCHANT}/>
+                        </Box>
+                    </RadioGroup>
+
+                    <Box className="terms-checkbox">
+                        <Checkbox
+                            checked={agreeChecked}
+                            onChange={handleAgreeCheckboxChange}
+                        />
+                        <Typography variant="body2">
+                            {AuthContent.TERMS}
+                        </Typography>
```

Comment:
roles and conditions are still with grey background.
Check yourself all of it. It's really annoying to run the app all the time to find such simple mistakes, it's your job